### DISCUSSION TITLE
Trim Paths

### DISCUSSION THREAD

**Discussion by tomgaa** [friction2d/friction#278] [General]
Is there a way to animate lines in Friction similar to the 'Trim Paths' feature in After Effects?

Attaching a video to show what I mean.

https://github.com/user-attachments/assets/73db5fa5-2d7b-4b87-b222-665eb52cfb9b

**Comment by pgilfernandez** [comment]
Yes, it can be done.
Check the second half of this video:

https://youtu.be/_a_j-s-C0Po?si=JvCD4OOqnxaMmWiT

The trick is to use subpath effect

**Reply by tomgaa** [reply]
Nice one! Can't believe I didn't find it myself 😅
Thanks @pgilfernandez!

**Reply by adambelis** [reply]
this  feature needs more discoverability 
https://github.com/friction2d/friction/issues/169
